### PR TITLE
Issues/#250 enable bips

### PR DESCRIPTION
**Pull request by macterra**
I found that enabling all the BIPs from the genesis block fixed the issue of the missing funds (unconfirmable txns)

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -292,11 +292,11 @@ class CTestNetParams : public CChainParams {
 #else  // BUILD_EQB
         consensus.nSubsidyAccelerationFactor = 1;
 #endif // END_BUILD
-        consensus.BIP16Height = 514; // 00000000040b4e986385315e14bee30ad876d8b47f748025b26683116d21aa65
-        consensus.BIP34Height = 21111;
+        consensus.BIP16Height = 0;
```
Comment (macterra) [inline src/chainparams.cpp:295]: P2SH

File: src/chainparams.cpp [hunk 2]
```diff
@@ -292,11 +292,11 @@ class CTestNetParams : public CChainParams {
...
+        consensus.BIP34Height = 0;
```
Comment (macterra) [inline src/chainparams.cpp:296]: Block v2, Height in Coinbase

File: src/chainparams.cpp [hunk 3]
```diff
@@ -292,11 +292,11 @@ class CTestNetParams : public CChainParams {
...
         consensus.BIP34Hash = uint256S("0x0000000023b3a96d3484e5abb3755c413e7d41500f8e2a5c3f0dd01299cd8ef8");
-        consensus.BIP65Height = 581885; // 00000000007f6655f22f98e72ed80d8b06dc761d5da09df0fa1dc4be4f861eb6
-        consensus.BIP66Height = 330776; // 000000002104c8c45e99a8853285a3b592602a3ccde2b832481da85e9e4ba182
+        consensus.BIP65Height = 0;
```
Comment (macterra) [inline src/chainparams.cpp:298]: OP_CHECKLOCKTIMEVERIFY

File: src/chainparams.cpp [hunk 4]
```diff
@@ -292,11 +292,11 @@ class CTestNetParams : public CChainParams {
...
+        consensus.BIP66Height = 0;
```
Comment (macterra) [inline src/chainparams.cpp:299]: Strict DER signatures

File: src/chainparams.cpp [hunk 5]
```diff
@@ -314,18 +314,18 @@ class CTestNetParams : public CChainParams {
...
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = 1199145601; // January 1, 2008
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = 1230767999; // December 31, 2008
+        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = Consensus::BIP9Deployment::ALWAYS_ACTIVE;
+        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
 
         // Deployment of BIP68, BIP112, and BIP113.
         consensus.vDeployments[Consensus::DEPLOYMENT_CSV].bit = 0;
-        consensus.vDeployments[Consensus::DEPLOYMENT_CSV].nStartTime = 1456790400; // March 1st, 2016
-        consensus.vDeployments[Consensus::DEPLOYMENT_CSV].nTimeout = 1493596800; // May 1st, 2017
+        consensus.vDeployments[Consensus::DEPLOYMENT_CSV].nStartTime = Consensus::BIP9Deployment::ALWAYS_ACTIVE;
```
Comment (macterra) [inline src/chainparams.cpp:322]: CHECKSEQUENCEVERIFY

**Review by r-soltani — APPROVED** [review]

**Review by ealefad — APPROVED** [review]

**Pull request merged**